src/main/java/games/strategy/engine/random/PlainRandomSource.java

```diff
@@ -1,35 +1,27 @@
 package games.strategy.engine.random;
 
+import static com.google.common.base.Preconditions.checkArgument;
+
+import javax.annotation.concurrent.GuardedBy;
+import javax.annotation.concurrent.ThreadSafe;
+
+import org.apache.commons.math3.random.MersenneTwister;
+
 /**
- * Gets random numbers from javas random number generators.
+ * A source of random numbers that uses a pseudorandom number generator.
  */
-public class PlainRandomSource implements IRandomSource {
-  /**
-   * Knowing the seed gives a player an advantage. Do something a little more clever than current time. which could
-   * potentially be guessed If the execution path is different before the first random call is made then the object will
-   * have a somewhat random address in the virtual machine, especially if a lot of UI and networking objects are created
-   * in response to semi-random mouse motion etc. if the execution is always the same then this may vary depending on
-   * the VM
-   */
-  private static long getSeed() {
```

Comment:
This method is no longer needed.  The Apache Commons implementation of Mersenne Twister uses an equivalent algorithm for the default seed.  Specifically, the default seed is based on an aggregate of the current time and the identity hash code of the `MersenneTwister` instance.